walk-in-interview/src/main/java/com/google/job/data/Requirement.java

```diff
@@ -0,0 +1,65 @@
+package com.google.job.data;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+
+import java.util.List;
+import java.util.Map;
+
+/** Enumeration that represents the requirements of jobs. */
+public enum Requirement {
+    // TODO(issue/26): add more requirements
+    O_LEVEL("o-level", ImmutableMap.of("en", "O Level")),
```

Comment:
Curious why we use train case here for the id while using all cap for the enum above?

Reply:
Ahh no special reason haha, since Riya define the string stable id first, then I just align with her